.github/workflows/build.yml

```diff
@@ -66,9 +66,10 @@ jobs:
           echo "GitHub event: $GITHUB_EVENT"
 
       - name: Checkout repo
-        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
+        uses: actions/checkout@a12a3943b4bdde767164f792f33f40b04645d846
 
-      - uses: actions/cache@937d24475381cd9c75ae6db12cb4e79714b926ed  # 2.1.7
+      - name: NPM cache
```

Comment:
Is this supposed to be named `NPM cache`? Also, I don't think we need an explicit action to cache NuGet packages as the `nuget/setup-nuget` action automatically caches.